cmake/maintainer.cmake

```diff
@@ -140,6 +140,7 @@ ENDIF()
 IF(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
   # Disable a few default Clang++ warnings
   STRING_APPEND(MY_CXX_WARNING_FLAGS " -Wno-null-conversion -Wno-unused-private-field")
+  STRING_APPEND(MY_CXX_WARNING_FLAGS " -Wno-missing-braces")
```

Comment:
Isn't it better to add braces to reported pieces of code instead of suppressing on whole code base?